### PR TITLE
API-4292: Export claim IDs for flashes and special issues

### PR DESCRIPTION
**Pull request by SlakrHakr**
## Description of change
Rake task for exporting established claims for a given datetime range. Primarily whether those claims contained any flashes or special issues.

## Original issue(s)
https://vajira.max.gov/browse/API-4292

## Things to know about this PR
Ran rake task locally.
Ran the logic within this rake task in production to get an initial report.

### CODE REVIEW

**Review by bastosmichael — CHANGES_REQUESTED** [review]
Test?

**Comment by SlakrHakr** [comment]
> Test?

https://github.com/department-of-veterans-affairs/vets-api/pull/5724/commits/b8b2dec2ac4094d8b8997dc549850b821435a8c4
https://github.com/department-of-veterans-affairs/vets-api/pull/5724/commits/c1459a10203b1641c30fb08dfd39b4b8361fc0d4

File: modules/claims_api/spec/lib/claims_api/power_of_attorney_pdf_constructor_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 require 'claims_api/power_of_attorney_pdf_constructor'
-require_relative '../support/pdf_matcher'
+require_relative '../../support/pdf_matcher'
```
Comment (SlakrHakr) [inline modules/claims_api/spec/lib/claims_api/power_of_attorney_pdf_constructor_spec.rb:5]: i dont know why i had to change this... this file was suddenly unable to find this pdf_matcher support file for some reason...

**Review by hinzed1127 — APPROVED** [review]
lgtm 👍

**Review by bastosmichael — APPROVED** [review]

**Pull request merged**